server/scala/api/src/main/scala/com/ctc/omega_edit/FFI.scala

```diff
@@ -17,11 +17,7 @@
 package com.ctc.omega_edit
 
 import com.ctc.omega_edit.api._
-import com.ctc.omega_edit.spi.{
-  NativeInfoNotFound,
-  PlatformInfoLoader,
-  VersionMismatch
-}
+import com.ctc.omega_edit.spi.{NativeInfoNotFound, PlatformInfoLoader, VersionMismatch}
```

Comment:
These came from running `sbt scalafmtAll`.

Reply:
So I believe you don't want to use that, you want to install `scalafmt-native` and run that. That is what is installed in the CI so that is just what I did at least.

Reply:
The CI Scala format check passed with these changes.  What exactly is wrong with `sbt scalafmtAll`?  There is also `sbt scalafmtCheckAll` for the check.